helix-term/src/commands/typed.rs

```diff
@@ -2075,8 +2075,9 @@ pub fn command_mode(cx: &mut Context) {
             // Handle typable commands
             if let Some(cmd) = typed::TYPABLE_COMMAND_MAP.get(parts[0]) {
                 let args = shellwords::shellwords(input);
+                let args = expand_args(cx.editor, &args[1..]);
```

Comment:
Should we expand arguments before determining shell words? That would eliminate the need to escaping spaces in args like `%sh{echo\ foo}`. What does kakoune do about this?

Reply:
done